gridscale/resource_gridscale_postgressql.go

```diff
@@ -0,0 +1,386 @@
+package gridscale
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/gridscale/gsclient-go/v3"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	errHandler "github.com/terraform-providers/terraform-provider-gridscale/gridscale/error-handler"
+
+	"log"
+)
+
+const postgresTemplateFlavourName = "postgres"
+
+const (
+	postgresReleaseValidationOpt = iota
+	postgresMaxCoreCountValidationOpt
+)
+
+func resourceGridscalePostgresSQL() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGridscalePostgresSQLCreate,
+		Read:   resourceGridscalePostgresSQLRead,
+		Delete: resourceGridscalePostgresSQLDelete,
+		Update: resourceGridscalePostgresSQLUpdate,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Description:  "The human-readable name of the object. It supports the full UTF-8 character set, with a maximum of 64 characters.",
+				Required:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+			"release_no": {
+				Type:         schema.TypeString,
+				Description:  "Release no. of postgresSQL service.",
+				Required:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
```

Comment:
We should make this `release` instead of `release_no`. The description should be `"The PostgreSQL release of this instance."`